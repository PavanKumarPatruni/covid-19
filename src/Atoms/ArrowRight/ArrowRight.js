import React, { forwardRef } from 'react';
import PropTypes from 'prop-types';

const ArrowRight = forwardRef(({ color, size }, ref) => {
  return (
    <svg
      ref={ref}
      xmlns="http://www.w3.org/2000/svg"
      width={size}
      height={size}
      viewBox="0 0 24 24"
      fill={color}
      stroke={color}
      transform="rotate(-90)"
      strokeWidth="2"
      strokeLinecap="round"
      strokeLinejoin="round"
    >
      <polyline points="6 9 12 15 18 9"></polyline>
    </svg>
  );
});

ArrowRight.propTypes = {
  color: PropTypes.string,
  size: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
};

ArrowRight.defaultProps = {
  size: 24,
  color: '#000000',
};

ArrowRight.displayName = 'ArrowUp';

export default ArrowRight;
