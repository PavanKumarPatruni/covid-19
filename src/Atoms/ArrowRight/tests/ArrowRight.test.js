import React from 'react';
import { shallow, mount } from 'enzyme';

import ArrowRight from '../ArrowRight';

describe('ArrowRight component', () => {
  const props = {
    color: '',
    size: 24,
  };

  test('match snapshot correctly', () => {
    const wrapper = shallow(<ArrowRight {...props} />);
    expect(wrapper).toMatchSnapshot();
  });

  it('match snapshot correctly', () => {
    const wrapper = mount(<ArrowRight {...props} />);
    expect(wrapper).toMatchSnapshot();
  });
});
