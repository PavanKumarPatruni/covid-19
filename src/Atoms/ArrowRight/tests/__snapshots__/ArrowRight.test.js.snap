// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ArrowRight component match snapshot correctly 1`] = `
<svg
  fill=""
  height={24}
  stroke=""
  strokeLinecap="round"
  strokeLinejoin="round"
  strokeWidth="2"
  transform="rotate(-90)"
  viewBox="0 0 24 24"
  width={24}
  xmlns="http://www.w3.org/2000/svg"
>
  <polyline
    points="6 9 12 15 18 9"
  />
</svg>
`;

exports[`ArrowRight component match snapshot correctly 2`] = `
<ArrowUp
  color=""
  size={24}
>
  <svg
    fill=""
    height={24}
    stroke=""
    strokeLinecap="round"
    strokeLinejoin="round"
    strokeWidth="2"
    transform="rotate(-90)"
    viewBox="0 0 24 24"
    width={24}
    xmlns="http://www.w3.org/2000/svg"
  >
    <polyline
      points="6 9 12 15 18 9"
    />
  </svg>
</ArrowUp>
`;
